Sources/SwiftDocC/SwiftDocC.docc/Resources/LinkableEntities.json

```diff
@@ -133,37 +142,45 @@
                         }
                     },
                     "kind": {
-                        "type": "string"
+                        "type": "string",
+                        "nullable": true
```

Comment:
I think these have always been incorrect. By default, OpenAPI doesn't expect values to be nullable. We explicitly set these values to null when they should be _removed_ as part of the patch application (i.e., the Swift version has an abstract but the Obj-C does not). However, if they are simply the same (the Swift version has the same abstract as the Obj-C version), the we don't encode them at all. 

So we can encode these as `null` _or_ we can not encode them at all and the distinction is important.